packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinSetupStrategies.tsx

```diff
@@ -1,94 +1,148 @@
+    const fee = (Number(account.balance) * 0.003).toString();
+
+    const reset = () => setStrategy('recommended');
+    const toggleConnectConfirmation = () => setConnectedConfirmed(!connectedConfirmed);
+    const toggleTermsConfirmation = () => setTermsConfirmed(!termsConfirmed);
+    const anonymize = () =>
+        actions.startCoinjoinSession(
+            account,
+            COINJOIN_STRATEGIES[strategy === 'custom' ? 'recommended' : strategy], // TODO: enable custom strategy
+        );
+
+    const tooltip = allChecked ? null : <Translation id="TR_ANONYMISATION_BUTTON_DISABLED" />;
 
     return (
-        <Wrapper>
-            <Content>
-                <Section style={{ flex: 1 }}>
-                    <SelectButton
-                        isActive={strategy === 'default'}
-                        variant="tertiary"
-                        onClick={() => setStrategy('default')}
-                    >
-                        Default
-                        {strategy === 'default' && <Icon icon="CHECK_ACTIVE" />}
-                    </SelectButton>
-                    <SelectButton
-                        isActive={strategy === 'speed'}
-                        variant="tertiary"
-                        onClick={() => setStrategy('speed')}
-                    >
-                        Speed
-                        {strategy === 'speed' && <Icon icon="CHECK_ACTIVE" />}
-                    </SelectButton>
-                </Section>
-                <Section>
-                    <CoinjoinSessionDetail account={account} {...selectedStrategy} />
-                </Section>
-            </Content>
-            <Button
-                onClick={() => actions.startCoinjoinSession(account, selectedStrategy)}
-                icon="ARROW_RIGHT_LONG"
-                alignIcon="right"
-            >
-                Start coinjoin
-            </Button>
-        </Wrapper>
+        <>
+            <StyledCard>
+                <Heading>
+                    <Translation id="TR_START_ANONYMISATION" />
+                </Heading>
+                <Row>
+                    <StyledCryptoAmountWithHeader
+                        header={
+                            <AmountHeading>
+                                <Translation id="TR_AMOUNT" />
+                            </AmountHeading>
+                        }
+                        value={account.balance}
+                        symbol={account.symbol}
+                    />
+                    <StyledCryptoAmountWithHeader
+                        header={
+                            <AmountHeading>
+                                <Translation id="TR_SERVICE_FEE" />
+                            </AmountHeading>
+                        }
+                        value={fee}
+                        symbol={account.symbol}
+                        note={<Translation id="TR_SERVICE_FEE_NOTE" />}
+                    />
+                </Row>
+                <Line />
```

Comment:
nit: not a fan of separator components, it's an extra StyledComponent for a situation when you can just use `border-bottom`. Another nit if you decide to keep it – name it `Separator`, please 😅

Reply:
You are right, I changed it here but I think it still makes sense in the box components (one of them has negative margin) where I just renamed it - 3770dcd2737b99b21f4c2d835c606a32271e4214